src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/statistikk/OppgavestatistikkTjeneste.kt

```diff
@@ -76,10 +79,23 @@ class OppgavestatistikkTjeneste(
     }
 
     private fun sendStatistikk(id: Long, tx: TransactionalSession) {
-        val (sak, behandling) = byggOppgavestatistikk(id, tx)
+        var (sak, behandling) = byggOppgavestatistikk(id, tx)
+        val erKode6 = runBlocking { pepClient.erSakKode6(sak.saksnummer) }
+        if (erKode6) {
+            sak = nullUtEventuelleSensitiveFelter(sak)
+            behandling = nullUtEventuelleSensitiveFelter(behandling)
+        }
         statistikkPublisher.publiser(sak, behandling)
     }
 
+    private fun nullUtEventuelleSensitiveFelter(sak: Sak): Sak {
```

Comment:
Det er en del jobb, men egentlig så burde vi si eksplisitt hvilke felter som er OK ... og kopiere disse inn. Dette fordi man her uforvarende kan sende med sensitiv informasjon hvis dette blir lagt til i kontrakten.

Jeg lurer også på om "aktorId" bør settes til "-5" på Sak

Reply:
Sitat @lfekete, ang aktorId på Sak:

"jeg tror ikke at det har mening at aktorid er skjermet, fordi da kan vi ikke identifisere det som kode6 relatert"

Reply:
Enig i at det hadde vært penere å eksplisitt kopiere lovlige felter, men er det verd å prioritere nå? Legge på TODO?

Reply:
OK, som førstesteg -- men fint om du lager en JIRA-oppgave for oppfølging.

Reply:
Oppfølging skrevet inn i TSF-3103